### ISSUE TITLE
Run RNA-Seq Workflows for APGI

### ISSUE THREAD

**Issue by b-f-chan** [icgc-argo/workflow-roadmap#271]
Reminder task to schedule the run for RNA-Seq workflows for the APGI program

**Comment by b-f-chan** [comment]
Postpone production run until @joneubank has returned and can provide support

**Comment by b-f-chan** [comment]
Next priority for Linda --> Get prod RDPC up and running again

**Comment by b-f-chan** [comment]
Model-Tee not ready yet for prod, Ummulkiram working on it with Linda

However, Linda and Edmund can work together to manually kick this off in prod and see if there's any hiccup in production (can also get Ummulkiram to help monitor log and debug issue quickly)

This week of Sept 14, we will kick off some jobs --> Any risk to kicking these off in prod w/o Model Tee?  Is there priority for this, this week?   No but just want to confirm the state of the environment

**Comment by lindaxiang** [comment]
Blocked by https://github.com/icgc-argo/workflow-management/issues/204

**Comment by lindaxiang** [comment]
Unblocked and 4 runs has been started and completed in Prod. Will keep to enqueue more after Collab comes back!

**Comment by b-f-chan** [comment]
Note:  Need ticket to track Ummulkiram's work for getting Model Tee into prod

**Comment by b-f-chan** [comment]
Put on hold because working group wants one new feature added to the alignment workflow first, so they will modify this first; we also have work to do as part of this request, see #287

**Comment by b-f-chan** [comment]
Blocker removed, we are good to process this run in prod; adding @edsu7 for assistance

**Comment by puneet-oicr** [comment]
performance issues noticed, 80 hours needed as opposed to the usual 2 hours.
UK and Henro looking in to this.

**Comment by puneet-oicr** [comment]
Seems to be working well for now.
Close the ticket in a week if no issues observed.

**Comment by b-f-chan** [comment]
80 hour issue is still present, likely some general problem with the cluster, instead of tracking here we'll use separate ticket as a blocker or dependency here
